specification/migrate/resource-manager/Microsoft.Migrate/AssessmentProjects/readme.md

```diff
@@ -0,0 +1,248 @@
+# Azure Migrate
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Azure Migrate.
+
+---
+
+### Java multi-api
+
+``` yaml $(java) && $(multiapi)
```

Comment:
Please put java configuration into readme.java.md like other languages.
Add JAVA SDK owner @XiaofeiCao for awareness.

Reply:
Thanks Yuchao.

Java doesn't use multiapi anymore, though we still want to have a separate readme.java.md. This seems first beta for Java SDK and we may want to disable model flattening in that file.

Reply:
@XiaofeiCao thanks! Can you please share any reference to similar file which has disabled model flattening? or any documentation that can give me an idea? Also, curious what does disabling model flattening helps us with ?

Reply:
> @XiaofeiCao thanks! Can you please share any reference to similar file which has disabled model flattening? or any documentation that can give me an idea? Also, curious what does disabling model flattening helps us with ?

Ah, I mean if you add readme.md.java, I could help add required settings for Java SDK there. 
NVM, let me extract java related settings, as the old template is out-of-date.

Reply:
Thanks @XiaofeiCao , please do share. Meanwhile let me commit the readme.java.md files.

Reply:
Just curious, why did other SDKs for Go threw no error?

Reply:
Hi @XiaofeiCao I have added the JAVA readme files. But Go SDK build still fails. Can you please check ?

Reply:
Thanks @srilipta-swain , appreciate it!

Looks like Go can't find input file, though the tag is there.. @tadelesh Could you help take a look?
I'll adjust Java settings and see we have the same issue.

Reply:
Got it, basically the default tag version's input & description was missing.

Reply:
@XiaofeiCao seems like your comment didn't show up earlier. I have fixed the Go SDK build, thanks!

Reply:
@XiaofeiCao I have addressed all the concerns, can you please confirm if I can proceed with the PR merge request ?

Reply:
Yes, please. Thanks!

Reply:
Thanks for confirmation, will proceed.